code/game/gamemodes/gameticker.dm

```diff
@@ -238,193 +240,179 @@ var/round_start_time = 0
-				switch( override )
-					if("nuclear emergency") //Nuke wasn't on station when it blew up
-						flick("intro_nuke",cinematic)
-						sleep(35)
-						world << sound('sound/effects/explosionfar.ogg')
-						flick("station_intact_fade_red",cinematic)
-						cinematic.icon_state = "summary_nukefail"
-					else
-						flick("intro_nuke",cinematic)
-						sleep(35)
-						world << sound('sound/effects/explosionfar.ogg')
-						//flick("end",cinematic)
-
-
-			if(2)	//nuke was nowhere nearby	//TODO: a really distant explosion animation
-				sleep(50)
-				world << sound('sound/effects/explosionfar.ogg')
-
-
-			else	//station was destroyed
-				if( mode && !override )
-					override = mode.name
-				switch( override )
-					if("nuclear emergency") //Nuke Ops successfully bombed the station
-						flick("intro_nuke",cinematic)
-						sleep(35)
-						flick("station_explode_fade_red",cinematic)
-						world << sound('sound/effects/explosionfar.ogg')
-						cinematic.icon_state = "summary_nukewin"
-					if("AI malfunction") //Malf (screen,explosion,summary)
-						flick("intro_malf",cinematic)
-						sleep(76)
-						flick("station_explode_fade_red",cinematic)
-						world << sound('sound/effects/explosionfar.ogg')
-						cinematic.icon_state = "summary_malf"
-					if("blob") //Station nuked (nuke,explosion,summary)
-						flick("intro_nuke",cinematic)
-						sleep(35)
-						flick("station_explode_fade_red",cinematic)
-						world << sound('sound/effects/explosionfar.ogg')
-						cinematic.icon_state = "summary_selfdes"
-					else //Station nuked (nuke,explosion,summary)
-						flick("intro_nuke",cinematic)
-						sleep(35)
-						flick("station_explode_fade_red", cinematic)
-						world << sound('sound/effects/explosionfar.ogg')
-						cinematic.icon_state = "summary_selfdes"
-				for(var/mob/living/M in living_mob_list)
-					if((M.loc.z in config.station_levels))
-						M.death()//No mercy
-		//If its actually the end of the round, wait for it to end.
-		//Otherwise if its a verb it will continue on afterwards.
-		spawn(300)
-			if(cinematic)	qdel(cinematic)		//end the cinematic
-			if(temp_buckle)	qdel(temp_buckle)	//release everybody
-		return
-
-
-	proc/create_characters()
-		for(var/mob/new_player/player in player_list)
-			if(player.ready && player.mind)
-				if(player.mind.assigned_role == "AI" || player.mind.special_role == "malfunctioning AI")
-					player.close_spawn_windows()
-					player.AIize()
-				else if(!player.mind.assigned_role)
-					continue
+//Plus it provides an easy way to make cinematics for other events. Just use this as a template
+/datum/controller/gameticker/proc/station_explosion_cinematic(station_missed = 0, override = null)
+	if(cinematic)
+		return	//already a cinematic in progress!
+
+	auto_toggle_ooc(1) // Turn it on
+	//initialise our cinematic screen object
+	cinematic = new /obj/screen{icon='icons/effects/station_explosion.dmi'; icon_state="station_intact"; layer=21; mouse_opacity=0; screen_loc="1,0";}(src)

```

Comment:
yeah it's a screen object, it's fine now that i look at it
still don't use curly braces, they are only useful if you plan to use initial() later, otherwise it's just gross